test/parse/yaml.go

```diff
@@ -34,6 +35,16 @@ kind: TaskRun
 	return &tr
 }
 
+// MustParseV1TaskRun takes YAML and parses it into a *v1.TaskRun
```

Comment:
Once we deprecate v1beta1 I think we should just modify MustParseTaskRun to use v1 by default. (Again, not necessary for this PR, but we should track this)